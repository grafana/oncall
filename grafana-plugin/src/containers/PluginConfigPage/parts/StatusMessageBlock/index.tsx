import React, { FC } from 'react';

import { Text } from 'components/Text/Text';

type Props = {
  text: string;
};

export const StatusMessageBlock: FC<Props> = ({ text }) => (
  <pre data-testid="status-message-block">
    <Text>{text}</Text>
  </pre>
);
