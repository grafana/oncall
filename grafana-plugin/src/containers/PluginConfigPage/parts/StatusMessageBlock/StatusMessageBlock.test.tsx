import React from 'react';

import { render } from '@testing-library/react';

import { StatusMessageBlock } from './StatusMessageBlock';

describe('StatusMessageBlock', () => {
  test('It renders properly', async () => {
    const component = render(<StatusMessageBlock text="helloooo" />);
    expect(component.baseElement).toMatchSnapshot();
  });
});
