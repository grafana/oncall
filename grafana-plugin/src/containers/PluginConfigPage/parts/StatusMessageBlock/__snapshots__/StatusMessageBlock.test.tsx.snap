// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusMessageBlock It renders properly 1`] = `
<body>
  <div>
    <pre
      data-testid="status-message-block"
    >
      <span
        class="css-77ouhj--undefined css-77ouhj--medium css-1287p17"
      >
        helloooo
      </span>
    </pre>
  </div>
</body>
`;
